tests/network-tests/start-storage.sh

```diff
@@ -0,0 +1,22 @@
+TMP=$0
+THIS_DIR=`dirname $TMP`
+
+# Run proposals tests first, since they require no leads hired
+$THIS_DIR/run-test-scenario.sh proposals
```

Comment:
Running proposals tests as part of `start-storage.sh` script makes it less general-purpose and reusable.

This is just a special case we need to take into account in `run-full-tests.sh`, as those (proposals) tests affect the hired leads.

Generally, if we expect the new leads hired to have `WORKER_ID=0`, we can setup a storage with:
```
set -a
. ../../.env
set +a

HOST_IP=`$THIS_DIR/get-host-ip.sh`
export COLOSSUS_1_URL="http://${HOST_IP}:3333"
export COLOSSUS_1_TRANSACTOR_KEY=$(docker run --rm --pull=always docker.io/parity/subkey:2.0.1 inspect ${COLOSSUS_1_TRANSACTOR_URI} --output-type json | jq .ss58Address -r)
export DISTRIBUTOR_1_URL="http://${HOST_IP}:3334"
$THIS_DIR/run-test-scenario.sh init-storage-and-distribution
docker-compose -f $THIS_DIR/../../docker-compose.yml up -d colossus-1
docker-compose -f $THIS_DIR/../../docker-compose.yml up -d distributor-1
```

Reply:
Amount of code in `start-storage.sh` was reduced. Proposals scenario is run in `run-full-tests.sh` only. Done.